README.md

```diff
@@ -9,16 +9,18 @@ SendinBlue's API matches the [OpenAPI v2 definition](https://www.openapis.org/).
 This library is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project and is reviewed and maintained by SendinBlue:
 
 - API version: 3.0.0
-- Package version: 7.x.x
+- Package version: 7.2.0
 - Build package: io.swagger.codegen.languages.JavascriptClientCodegen
+For more information, please visit [https://account.sendinblue.com/support](https://account.sendinblue.com/support)
 
 ## Installation
 
 ### For [Node.js](https://nodejs.org/)
 
 #### npm
 
-The following recommended installation requires [npm](https://npmjs.org/). If you are unfamiliar with npm, see the [npm docs](https://npmjs.org/doc/).
+To publish the library as a [npm](https://www.npmjs.com/),
```

Comment:
These 2 lines need to stay as they were before
`The following recommended installation requires [npm](https://npmjs.org/). If you are unfamiliar with npm, see the [npm docs](https://npmjs.org/doc/).`